doc/world_builder_declarations.schema.json

```diff
@@ -1168,7 +1180,11 @@
                                                                             "items": {
                                                                                 "type": "array",
                                                                                 "minItems": 1,
+<<<<<<< HEAD
```

Comment:
This doesn't look correct. Can you run `ctest -R generate_declarations_and_schema` and check in the new result?

Reply:
I fixed this. But this is from the conflict I get from rebase. I am not sure why this happens:

Auto-merging doc/world_builder_declarations.schema.json
CONFLICT (content): Merge conflict in doc/world_builder_declarations.schema.json
Auto-merging doc/world_builder_declarations_closed.md
CONFLICT (content): Merge conflict in doc/world_builder_declarations_closed.md
Auto-merging doc/world_builder_declarations_open.md
CONFLICT (content): Merge conflict in doc/world_builder_declarations_open.md